---
title: Seasonal influenza circulation as of May 31, 2022
authors: "John Huddleston (Bedford Lab)"
date: "June 3, 2022"
dataset: "https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full"
abstract: Global responses to the SARS-CoV-2 pandemic had a strong effect on seasonal influenza circulation. A/H3N2 circulation has returned to pre-pandemic levels and global circulation driven by the success of 2a.2 and its subclades. A/H1N1pdm remains at lower levels of circulation since the pandemic and its genetic diversity exhibits strong geographic structure. B/Vic populations in China and the rest of the world are strongly isolated. B/Yam appears to be extinct. We observe what are effectively two meta-populations of seasonal influenza where a) individuals in China (~18% of the global population) may no longer encounter H3N2 or H1N1pdm but may experience consistent Vic circulation and b) individuals in the rest of the world encounter H3N2, H1N1pdm, and antigenically distinct Vic clades from those circulating in China. Long branches with limited geographic spread for all major lineages indicate continuous circulation of seasonal influenza in many regions of the world.

---

# [Seasonal influenza circulation since 2018](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

To understand global seasonal influenza circulation, we build phylogenies of hemagglutinin (HA) and neuraminidase (NA) using data submitted to [GISAID](http://gisaid.org/).
Seasonal influenza consists of four major lineages including:

 - A/H3N2 ([HA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra), [NA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/na/2y/cell/fra))
 - A/H1N1pdm ([HA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi), [NA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/na/2y/cell/hi))
 - B/Victoria (Vic) ([HA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi), [NA tree](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/na/2y/cell/hi))
 - B/Yamagata (Yam) ([HA tree](https://nextstrain.org/flu/seasonal/yam/ha/2y), [NA tree](https://nextstrain.org/flu/seasonal/yam/na/2y))

```auspiceMainDisplayMarkdown
<div>
    <img src="https://raw.githubusercontent.com/huddlej/seasonal-flu-reports/main/figures/2022-05-31-seasonal-influenza-circulation-since-2018.png" width="100%" />
</div>

The global response to the SARS-CoV-2 pandemic substantially reduced seasonal influenza circulation following April 2020.
The B/Yamagata lineage has not been observed since March 2020, indicating that the pandemic might have eliminated one of the major influenza lineages.
Only A/H3N2 appears to have returned to pre-pandemic levels of circulation.
```

# [Viruses accumulate mutations along transmission trees](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

Transmission trees represent the actual infection of individuals and subsequent onward transmission from each infection.
We can sample the genomes of some (often small) proportion of all infections.
Viral genomes accumulate mutations during replication, allowing us to infer the transmission tree.

```auspiceMainDisplayMarkdown
<div>
    <img src="https://docs.nextstrain.org/en/latest/_images/infection_tree_combined.png" width="100%" />
</div>

See [Nextstrain's guide on how to read a tree](https://docs.nextstrain.org/en/latest/learn/interpret/how-to-read-a-tree.html), for more details.
```

# [Phylogenies estimate the original transmission tree from sampled genomes](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

Phylogenetic trees represent the inferred relationships between sampled virus genomes.
Phylogenetic algorithms rely on the shared mutations present in the sampled genomes to infer these relationships.

```auspiceMainDisplayMarkdown
<div>
    <img src="https://docs.nextstrain.org/en/latest/_images/toy_alignment_tree.png" width="100%" />
</div>

See [Nextstrain's guide on how to read a tree](https://docs.nextstrain.org/en/latest/learn/interpret/how-to-read-a-tree.html), for more details.
```

# [Clades represent potentially relevant biological or epidemiological groups ](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

Viral genomes that share a single common ancestor in a phylogeny are called "clades".
We define a clade by the mutations that we find in its inferred ancestral genome.
These clades may represent viruses with similar phenotypes or geographic distributions.

```auspiceMainDisplayMarkdown
<div>
    <img src="https://docs.nextstrain.org/en/latest/_images/introductions.png" width="100%" />
</div>

Each blue subtree in the left and center panels of the figure above represents a clade in the corresponding tree.
See [Nextstrain's guide on how to read a tree](https://docs.nextstrain.org/en/latest/learn/interpret/how-to-read-a-tree.html), for more details.
```

# [H3N2 diversity in the last four years](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

At the beginning of the pandemic, most H3N2 clades descended from the 3C.2a1b clade.
The clade 3C.3a is a notable exception, a distantly related sibling clade that had circulated at low frequency since late 2013 (7 years).
Most of the clades circulating as of January 2020 were replaced by a new clade 2a.2.

# [H3N2 circulation dropped following global pandemic responses](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?branchLabel=none&d=tree&dmin=2020-01-01&p=full)

In January 2020, all major clades circulated globally.
By April 2020, almost all clades had disappeared from genome surveillance data.
The reappearance of major clades later in 2020 indicates ongoing, unobserved transmission despite global pandemic responses.

# [H3N2 circulation during the pandemic shows regional structure](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,map&dmin=2020-04-01&p=grid&r=region&transmissions=hide)

H3N2 clades show limited geographic distributions throughout the pandemic.
For example, Africa, Southeast Asia, and South America all exhibit distinct patterns of clades circulating in the last two years.

# [Africa experienced distinct H3N2 clades from the rest of the world](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,map&dmin=2020-04-01&f_region=Africa&p=grid&transmissions=hide)

The 2a1b subclades 1a and 1b almost exclusively circulated in Africa, especially in West Africa.
However, we also observe distinct genomic structure between countries within the same regions of Africa.

# [South and Southeast Asia experienced distinct clades from each other](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,map&dmin=2020-04-01&f_region=China,South%20Asia,Southeast%20Asia,Japan%20Korea&p=grid&transmissions=hide)

Adjacent countries in Southeast Asia experienced the same clade, 2a.1, while countries in South Asia did not observe this clade at all.
Instead, we see the success of the 2a.2 clade that has gone on to dominate globally.
Notably, China experienced completely different clades and has not reported any H3N2 since July 2020.

# [Even within the United States we observe some structured H3N2 circulation](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,map&dmax=2022-05-31&dmin=2020-04-01&f_country=Usa&p=grid&r=division&transmissions=hide)

Although the new 53G subclade clearly dominates most states in the United States, other clades like 53N and 2a.2 appear in distinct regional clusters.

# [Successful H3N2 clades carry RBS-adjacent mutations](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?c=rb&d=tree&p=full&transmissions=hide)

The successful H3N2 clade 2a.2 and its descendants acquired three mutations adjacent to the receptor binding site (RBS).

# [2a.2 clade lost a glycosylation site associated with 3c.2A for 8 years](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?c=gt-HA1_160&d=tree&p=full&transmissions=hide)

The 160I mutation removes a glycosylation site that has shielded other epitopes in 3C.2a strains for the last eight years ([Zost et al. 2017](https://www.pnas.org/doi/10.1073/pnas.1712377114)).

# [Models based on ferret serology show antigenic advance in 2a.2](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?c=cTiterSub&d=tree&p=full&transmissions=hide&label=clade:3C.2a1b.2a.2&branchLabel=aa)

Models trained on neutralization assays (FRA and HINT assays) with ferret sera show antigenic advance of 2a.2 and its descendants relative to sibling clades.

# [Forecasting models predict the composition of future H3N2 populations](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?d=tree,frequencies&p=full)

We estimate the fitness of currently circulating strains using biologically-relevant predictors like antigenic advance ([Neher at al. 2016](https://bedford.io/papers/neher-titers/)) or rapid recent population growth ([Neher et al. 2014](https://elifesciences.org/articles/03568)) and project the frequencies of these strains forward one year in the future to estimate the composition of the future population.
We calculate the Hamming distance in nucleotide differences between each recent strain and the future population to find current strains that best represent the future population.

```auspiceMainDisplayMarkdown
<div>
    <img src="https://bedford.io/images/blog/flu-forecasting-distance-based-fitness-model.png" width="100%" />
</div>

See [Huddleston et al. 2020](https://bedford.io/papers/huddleston-flu-forecasting/), for more details.
```

# [Forecasting models predict continued success of 53G and 53N](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h3n2/ha/2y/cell/fra?branches=hide&d=tree,frequencies&l=scatter&p=full&scatterX=weighted_distance_to_future_by_cTiter_x-ne_star&scatterY=weighted_distance_to_future_by_ne_star-lbi&transmissions=hide)

Models based on either antigenic data or rapid growth in the phylogeny both agree in their predictions that 53G and 53N clades will continue to circulate in the next year.
Samples in 53G, in particular, are genetically closer to the predicted future population.

# [H1N1pdm diversity in the last four years](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi?d=tree,frequencies&p=full)

The H1N1pdm clade 6B.1A represents the circulating diversity in the last four years.
The subclade 6B.1A.5, in particular, was dominating at the beginning of the pandemic.
Since the pandemic, its subclades 5a.1 and 5a.2 have nearly replaced all other clades.

# [H1N1pdm clades show strong geographic structure](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi?d=tree,map&dmin=2020-04-01&label=clade:6B.1A.5a&p=grid&r=region&transmissions=hide)

Since April 2020, we see only regional circulation of H1N1pdm clades.
Africa and Europe primarily experienced 5a.1, while South Asia, West Asia, and Oceania experienced 5a.2.
Notably, China has not reported any H1N1pdm since December 2020.
A sibling clade of 5a.1 and 5a.2 (called 5a, here) has continued to circulate in Africa alone.

# [5a.2 appeared to have died out until its reemergence in Southeast Asia in mid-2021](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi?c=region&d=tree,frequencies&p=full&r=region&transmissions=hide&label=clade:6B.1A.5a)

During the pandemic, H1N1pdm initially seemed to disappear and then only appeared to be circulating in Africa as 5a.1.
In the middle of 2021, however, 5a.2 reemerged in Southeast Asia and eventually circulated more globally.

# [5a.2 has not been growing recently despite its antigenic advance](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi?c=cTiterSub&d=tree,frequencies&p=full&r=region&transmissions=hide)

Models trained on hemagglutination inhibition (HI) assays with ferret sera show strong antigenic advance for 5a.2.
However, this clade has not been successful recently and appears to be outcompeted by 5a.1.

# [Most recently successful 5a.1 clades have reassorted with a different NA background](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/ha/2y/cell/hi:community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/h1n1pdm/na/2y/cell/hi?d=tree&f_clade_membership=6B.1A.5a.1&p=full&r=region&transmissions=hide)

The most recent strains of 5a.1 descend from an HA clade that reassorted with an NA from a novel background.
This reassortment appears to have happened in Africa.

# [Vic diversity in the last four years](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi?d=tree,frequencies&p=full)

In the last four years, a Vic clade, V1A.3, carrying a triple deletion (HA1 163-165) replaced all other circulating clades at the time.
At the beginning of the pandemic, V1A.3's sweep was nearly complete, and a subclade, V1A.3a had already begun to emerge.
Since the pandemic, this subclade diversified into V1A.3a1 and V1A.3a2 and nearly replaced its sibling clades from V1A.3.

# [As with other lineages, Vic has shown strong geographic structure during the pandemic](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi?d=tree,map&dmin=2020-04-01&label=clade:V1A.3&p=grid&r=region&transmissions=hide)

The 3a.2 subclade has circulated globally, but it has dominated primarily in Europe, Africa, and South and Southeast Asia.
Interestingly, 3a.1 has almost exclusively circulated in China, since April 2020, with no onward transmission globally.
Also somewhat surprisingly, other subclades of V1A.3 have continued to circulate in China, North America, South America, Africa, and Europe.

# [Long-branched subclades in V1A.3 show limited geographic spread](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi?c=region&d=tree&f_clade_membership=V1A.3&f_epiweek=202135,202136,202137,202138,202139,202140,202141,202142,202143,202144,202145,202146,202147,202148,202149,202150,202151,202152,202201,202202,202203,202204,202205,202206,202207,202208,202209,202210,202211,202213,202214,202215,202216,202217,202134&p=full)

Recent subclades of V1A.3 primarily originate from a single region of the world, indicating limited geographic spread of these clades and continuous circulation within a given region.

# [China has experienced continued co-circulation of V1A.3 subclades](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi?d=tree,map&dmin=2020-04-01&f_region=China&p=grid&r=division&transmissions=hide)

In China, we see continued co-circulation of V1A.3 subclades including 3a.1 and a subclade of 3a.2 defined by a HA1:122Q mutation.
These latter two clades appear in nearly every division in China.
These clades most likely represent the diversity of Vic circulating when quarantine policies began.
Since the beginning of the pandemic, no new Vic strains have appeared in China from the rest of the world.
Vic may now be the only seasonal influenza circulating in China.

# [Samples from China's 122Q subclade have transmitted outside of the country](https://nextstrain.org/community/huddlej/seasonal-flu-reports/flu/seasonal/2022-05-31/vic/ha/2y/cell/hi?c=region&d=tree,map&gt=HA1.122Q,nuc.1806A&label=clade:V1A.3a.2&p=grid&transmissions=hide)

Although China's quarantine policies prevent the transmission of new variation into the country, they do not limit transmission from China to the rest of the world.
Recently collected strains in Montana, USA and Sydney, Australia from China's 122Q subclade of 3a.2 show the first observed cases of this onward transmission.

# [B/Yamagata has not been observed since March 2020](https://nextstrain.org/flu/seasonal/yam/ha/2y?d=tree&p=full)

Despite active global surveillance efforts, no verified sample of B/Yamagata has been reported since March 2020.
Although patterns from the other major lineages show how influenza can continue to circulate undetected in the same region and in the absence of global travel, the lack of observed Yam samples after this long suggests that the lineage may be extinct.

# [Conclusions](https://nextstrain.org/flu/seasonal/yam/ha/2y)

```auspiceMainDisplayMarkdown
## Lessons learned about seasonal influenza from the SARS-CoV-2 pandemic era

 - Global responses to the SARS-CoV-2 pandemic had a strong effect on seasonal influenza circulation.
 - A/H3N2 circulation has returned to pre-pandemic levels and global circulation driven by the success of 2a.2 and its subclades.
 - A/H1N1pdm remains at lower levels of circulation since the pandemic and its genetic diversity exhibits strong geographic structure.
 - B/Vic populations in China and the rest of the world are strongly isolated.
 - B/Yam appears to be extinct.
 - We observe what are effectively two meta-populations of seasonal influenza where a) individuals in China (~18% of the global population) may no longer encounter H3N2 or H1N1pdm but may experience consistent Vic circulation and b) individuals in the rest of the world encounter H3N2, H1N1pdm, and antigenically distinct Vic clades from those circulating in China.
 - Long branches with limited geographic spread for all major lineages indicate continuous circulation of seasonal influenza in many regions of the world.
```
